web/src/api/hooks/revisions.ts

```diff
@@ -0,0 +1,95 @@
+// temporary_disabled_rules
+/* eslint-disable react-hooks/rules-of-hooks */
+import {
+    FilterWithDocumentExtraOption,
+    Operators,
+    PagedResponse,
+    QueryHookParamsType,
+    QueryHookType,
+    Revision,
+    SearchBody,
+    SortingDirection
+} from 'api/typings';
+import { useQuery } from 'react-query';
+import { pageSizes } from 'shared/primitives';
+import { useBadgerFetch } from './api';
+
+const namespace = process.env.REACT_APP_CATEGORIES_API_NAMESPACE;
+
+export const useRevisions: QueryHookType<QueryHookParamsType<Revision>, PagedResponse<Revision>> = (
+    params,
+    options
+) => {
+    const { searchText, sortConfig, size: pageSize, page: pageNum, filters } = params;
+    return useQuery(
+        ['revisions', searchText, sortConfig, pageSize, pageNum],
+        () => revisionsFetcher(pageNum, pageSize, searchText, sortConfig, filters),
+        options
+    );
+};
+export function revisionsFetcher(
+    page = 1,
+    size = pageSizes._15,
+    searchText?: string | null,
+    sortConfig: {
+        field: keyof Revision;
+        direction: SortingDirection;
+    } = {
+        field: 'revision',
+        direction: SortingDirection.ASC
+    },
+    filters: FilterWithDocumentExtraOption<keyof Revision>[] = []
+): Promise<PagedResponse<Revision>> {
+    if (searchText) {
+        filters.push({
+            field: 'revision',
+            operator: Operators.ILIKE,
+            value: `%${searchText.trim().toLowerCase()}%`
```

Comment:
you utilize ILIKE operator which is case insensitive. You can omit toLowerCase method